README.md

```diff
@@ -21,17 +21,20 @@ The table below lists models that were open sourced by official sector entities
 | USA | Federal Reserve Bank of New York (FRBNY) | | Dynamic stochastic general equilibrium (DSGE) | https://github.com/FRBNY-DSGE/DSGE.jl | Julia | BSD-3-Clause License | [Blog post](https://libertystreeteconomics.newyorkfed.org/2015/12/the-frbny-dsge-model-meets-julia/) |
 | USA | Federal Reserve Board | FRB/US | Large-scale, general equilibrium model | https://www.federalreserve.gov/econres/us-models-package.htm | EViews | Same as for the website | https://www.federalreserve.gov/econres/us-models-about.htm |
 | USA | Federal Reserve Board | Estimated Dynamic Optimization (EDO) Model | DSGE | https://www.federalreserve.gov/econres/edo-model-package.htm | Matlab/Octave + Dynare | Same as for the website | https://www.federalreserve.gov/econres/edo-models-about.htm |
+| USA | [Policy Simulation Library](https://pslmodels.org/) | [`OG-USA`](https://github.com/PSLmodels/OG-USA) | Large-scale overlapping generations model | https://github.com/PSLmodels/OG-USA | Python | [CC0 1.0 Universal Summary](https://github.com/PSLmodels/OG-USA/blob/master/LICENSE.md) | Large-scale overlapping generations model of U.S. fiscal policy. It uses as a dependency the [`OG-Core`](https://github.com/PSLmodels/OG-Core) model. `OG-USA` is currently set up to inteface with the open-source [`Tax-Calculator`](https://github.com/PSLmodels/Tax-Calculator) microsimulation model. However, it has been used with other microsimulation models as well (e.g., Tax Policy Center microsimulation model). [`OG-USA` documentation](https://pslmodels.github.io/OG-USA/) is thorough and up-to-date. |
+| General | [Policy Simulation Library](https://pslmodels.org/) | [`OG-Core`](https://github.com/PSLmodels/OG-Core) | Large-scale overlapping generations model | https://github.com/PSLmodels/OG-Core | Python | [CC0 1.0 Universal Summary](https://github.com/PSLmodels/OG-Core/blob/master/LICENSE.md) | This core model can be run on its own. But more effective is to create a country-specific calibration that uses `OG-Core` as a dependency (e.g., [`OG-USA`](https://github.com/PSLmodels/OG-USA) and [`OG-UK`](https://github.com/PSLmodels/OG-UK)). [`OG-Core` documentation](https://pslmodels.github.io/OG-Core/) is thorough and up-to-date. |
 
 ## Other related resources
 
 Below are other resources that are worth consulting for their wealth of model code.
```

Comment:
Transposing the models added in the original PR to a new table (with the same design) under this section, as mentioned in the comments above. The models are ordered from general to country-specific, and alphabetically within each country.

```suggestion
Below are other resources that are worth consulting for their wealth of model code. The table below lists policy models open sourced by entities not in the official sector. Other sources are listed after the table.

| Jurisdiction | Institution | Model name | Model type | Link to code | Programming language | License | About |
|---|---|---|---|---|---|---|---|
| General | [Policy Simulation Library](https://pslmodels.org/) | [`OG-Core`](https://github.com/PSLmodels/OG-Core) | Large-scale overlapping generations model | https://github.com/PSLmodels/OG-Core | Python | [CC0 1.0 Universal Summary](https://github.com/PSLmodels/OG-Core/blob/master/LICENSE.md) | This core model can be run on its own. But more effective is to create a country-specific calibration that uses `OG-Core` as a dependency (e.g., [`OG-USA`](https://github.com/PSLmodels/OG-USA) and [`OG-UK`](https://github.com/PSLmodels/OG-UK)). [`OG-Core` documentation](https://pslmodels.github.io/OG-Core/) is thorough and up-to-date. |
| UK | [Policy Simulation Library](https://pslmodels.org/) | [`OG-UK`](https://github.com/PSLmodels/OG-UK) | Large-scale overlapping generations model | https://github.com/PSLmodels/OG-UK | Python | [CC0 1.0 Universal Summary](https://github.com/PSLmodels/OG-UK/blob/main/LICENSE.md) | This model is currently under development. Large-scale overlapping generations model of U.K. fiscal policy. It uses as a dependency the [`OG-Core`](https://github.com/PSLmodels/OG-Core) model logic and interfaces with the [`OpenFisca-UK`](https://policyengine.github.io/openfisca-uk/) open-source microsimulation model. [`OG-UK` documentation](https://pslmodels.github.io/OG-UK/) is also under development. |
| USA | [Policy Simulation Library](https://pslmodels.org/) | [`OG-USA`](https://github.com/PSLmodels/OG-USA) | Large-scale overlapping generations model | https://github.com/PSLmodels/OG-USA | Python | [CC0 1.0 Universal Summary](https://github.com/PSLmodels/OG-USA/blob/master/LICENSE.md) | Large-scale overlapping generations model of U.S. fiscal policy. It uses as a dependency the [`OG-Core`](https://github.com/PSLmodels/OG-Core) model. `OG-USA` is currently set up to inteface with the open-source [`Tax-Calculator`](https://github.com/PSLmodels/Tax-Calculator) microsimulation model. However, it has been used with other microsimulation models as well (e.g., Tax Policy Center microsimulation model). [`OG-USA` documentation](https://pslmodels.github.io/OG-USA/) is thorough and up-to-date. |
```